generators/bdk/java/templates/ext-app/application.yaml.ejs

```diff
@@ -0,0 +1,43 @@
+# BDK Core Configuration
+bdk:
+  host: <%= host %>
+  bot:
+    username: <%= username %>
+    privateKey:
+      path: <%= privateKeyPath %>
+
+  app:
+    appId: <%= appId %>
+    privateKey:
+      path: <%= privateKeyPath %>
+
+# BDK App Configuration
+bdk-app:
+  auth:
+    enabled: true
+    jwtCookie:
+      enabled: true
+      maxAge: 1d
+  cors:
+    "[/**]":
+      allowed-origins: "*"
+      allowed-credentials: false
+  tracing:
+    enabled: true
+    urlPatterns:
```

Comment:
I think we can go with - separated properties, I tested it and it seems to work with both formats